package.json

```diff
@@ -43,12 +44,13 @@
   },
   "files": [
     "binding.gyp",
+    "browser.js",
+    "index.d.ts",
+    "index.js",
```

Comment:
Potentially, this aren't needed, but [the `files` documentation](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#files) only mentions `main`, and I think it's better to just include them all...